remoteit/desktop
# DESK-972 increment port assignments

### Description


Fixing auto-assignment of next free port

### Test plan



1. Sign in
2. Go to a new service
3. Click connect
4. You should see a free port assigned
5. Go to another new service
6. You should see the next available port assigned

### Ticket



<https://remoteit.atlassian.net/browse/DESK-972>
